var/spack/repos/builtin/packages/comd/package.py

```diff
@@ -37,20 +36,72 @@ class Comd(MakefilePackage):
     versions of CoMD will be released to incorporate the lessons learned from
     the co-design process."""
 
-    homepage = "http://exmatex.github.io/CoMD/"
+    tags = ['proxy-app']
 
-    version('master', git='https://github.com/exmatex/CoMD.git',
-            branch='master')
+    homepage = "http://www.exmatex.org/comd.html"
+    url      = "https://github.com/exmatex/CoMD/archive/master.tar.gz"
 
-    depends_on('mpi')
+    version('master', git='https://github.com/exmatex/CoMD.git', branch='master')
 
-    build_directory = 'src-mpi'
```

Comment:
What you can do is:
```python
@property
def build_directory(self):
    if '+mpi' in self.spec:
        return 'src-mpi'
    else:
        return 'src-openmp'
```
Then you can refer to this in the rest of the file as `self.build_directory`. You won't need to set `--directory` anywhere either. You'll also want to add:
```python
conflicts('~mpi', '~openmp')
conflicts('+mpi', '+openmp')
```
so that it's impossible to choose both or neither.